app/modules_test.go

```diff
@@ -1,4 +1,5 @@
 // +build experimental
```

Comment:
This is more coming from https://github.com/regen-network/regen-ledger/pull/322 but this means now that the `testutil/network` tests are now only running in experimental mode, which weren't the case and doesn't really make sense actually, should we have a separate test file for this instead?